newIDE/app/src/EventsSheet/ParameterFields/GenericExpressionField/FormatExpressionCall.js

```diff
@@ -14,7 +14,8 @@ const filterOutCodeOnlyParameters = (
 
     return (
       index < parametersCount &&
-      !expressionMetadata.getParameter(index).isCodeOnly()
+      !expressionMetadata.getParameter(index).isCodeOnly() &&
```

Comment:
You've added your changes in a function named "filterOutCodeOnlyParameters".
What does this name implies? What does it describe? Is this name still adapted after you add your changes?
Better: could you make your changes in a new function, with a name describing what you're doing? :)

Reply:
I tried to solve w/o creating a function, but I'll do that

Reply:
There are 3 conditions and all of them uses filterOutCodeOnlyParameters, not sure if I should make a more abstract function that handles all cases.

Reply:
You can make a `filterParameters` that calls the necessary functions (filterOutCodeOnlyParameters and the new one)